### ISSUE TITLE
question about the format of the top3 scores in the precomputed dataset.

### ISSUE THREAD

**Issue by WxWstranger** [vpulab/Semantic-Aware-Scene-Recognition#14]
First of all, thanks a lot for sharing this great project. I want to use this model to predict the scenes on COCO dataset, however, I'm confused about the format of the top3 scores in the precomputed dataset. Why is the highest score of a pixel about 0.4 but 1. what function do you use to get these scores? 

In  https://github.com/CSAILVision/semantic-segmentation-pytorch the scores are product by softmax.

Thanks a lot!

**Comment by alexlopezcifuentes** [comment]
Hi! This question is further discussed in the paper, in case you want to take a look.

In summary, if you have an RGB image as input for the Semantic Segmentation Network you will end up having a probability distribution of semantic scores **for each pixel**. This means that if you have 120 semantic classes, for each pixel you will have a distribution of 120 scores. The sum of all that scores must be 1 (probability distribution) but this does not mean that the top score must be 1. You can have 120 scores summing up to 1 with the highest one being 0.4.

**Comment by WxWstranger** [comment]
> Hi! This question is further discussed in the paper, in case you want to take a look.
> 
> In summary, if you have an RGB image as input for the Semantic Segmentation Network you will end up having a probability distribution of semantic scores **for each pixel**. This means that if you have 120 semantic classes, for each pixel you will have a distribution of 120 scores. The sum of all that scores must be 1 (probability distribution) but this does not mean that the top score must be 1. You can have 120 scores summing up to 1 with the highest one being 0.4.

Thanks for your quick reply!

So I think the scores is simply got by Softmax like the implement https://github.com/CSAILVision/semantic-segmentation-pytorch.

However, when I run the evaluate.py on MITIndoor dataset, I checked the top3 scores of a pixel of the first sample, I got the following data:
![2020-03-27_183844](https://user-images.githubusercontent.com/22026000/77748108-cb654280-705a-11ea-8fd0-077e99b079fb.png)

The largest score is about 0.4, the 2nd and 3rd large scores are 0. Is that means the sum of all the scores is not 1? or I misunderstand somewhere？
